packages/valory/agents/oracle/aea-config.yaml

```diff
@@ -138,23 +138,13 @@ public_id: valory/p2p_libp2p_client:0.1.0
 type: connection
 config:
   nodes:
-  - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9005}
-    public_key: ${P2P_PUBLIC_KEY:str:02d3a830c9d6ea1ae91936951430dee11f4662f33118b02190693be835359a9d77}
-  - uri: ${P2P_URI:str:acn.staging.autonolas.tech:9006}
-    public_key: ${P2P_PUBLIC_KEY:str:02e741c62d706e1dcf6986bf37fa74b98681bc32669623ac9ee6ff72488d4f59e8}
+  - uri: localhost:11000
```

Comment:
why change to local? Default should be remote?

Reply:
The same reason as we keep URLs for chain or other specs to some local URLs, The user will mostly run these agents in a local environment and the deployments will be in once a moon so I think it's not a good idea to point these to a production or a staging node. 

These were supposed to be overridable env vars which we use when deploying the service, But because of this https://github.com/valory-xyz/open-autonomy/pull/1432#pullrequestreview-1142255156 we cannot do it.

Reply:
`${P2P_URI:str:localhost:11000}`